chia/cmds/wallet.py

```diff
@@ -75,8 +75,13 @@ def get_transactions_cmd(wallet_rpc_port: int, fingerprint: int, id: int, offset
     required=True,
 )
 @click.option("-t", "--address", help="Address to send the XCH", type=str, required=True)
-def send_cmd(wallet_rpc_port: int, fingerprint: int, id: int, amount: str, fee: str, address: str) -> None:
-    extra_params = {"id": id, "amount": amount, "fee": fee, "address": address}
+@click.option(
+    "-c", "--confirm", help="Submits transaction without checking for unusual values", is_flag=True, default=False
```

Comment:
I think calling this "confirm" may be a bit unfortunate, since that's already a term in blockchains relating to transactions.
Not setting this could be interpreted as "submit a transaction, but don't confirm it".

The best term I can think of for this is "fat finger", but it's not great either.

maybe `-f`, `--force` would make sense.

Reply:
-f is already in use for fingerprint